distribution/docker/build.gradle

```diff
@@ -38,13 +38,12 @@ ext.expansions = { architecture, oss, local ->
   final String classifier = "aarch64".equals(architecture) ? "linux-aarch64" : "linux-x86_64"
   final String elasticsearch = oss ? "elasticsearch-oss-${VersionProperties.elasticsearch}-${classifier}.tar.gz" : "elasticsearch-${VersionProperties.elasticsearch}-${classifier}.tar.gz"
   return [
-    'base_image'          : "aarch64".equals(architecture) ? "arm64v8/centos:7" : "centos:7",
+    'base_image'          : "centos:7",
```

Comment:
@jasontedor Question for you, since `centos` uses manifests for their Docker builds pulling from the specific arch is unneeded. Was there another reason to directly pull from `arm64v8/centos:7`? Or would pulling from `centos:7` work? You can see the manifests/archs that Centos supports here, https://hub.docker.com/_/centos?tab=tags.

Reply:
The thinking here is that using an explicit base image makes it possible to cross-build the ARM image, for example using Docker for Mac. It's entirely possible that we'll revert to just specifying `centos:7` in the future however.

Reply:
> The thinking here is that using an explicit base image makes it possible to cross-build the ARM image, for example using Docker for Mac.

Makes perfect sense! I went a head a made the `amd64` image explicit too in case you’re not on a `x86` system. 

> It's entirely possible that we'll revert to just specifying centos:7 in the future however.

Using manifests for this would help simplify things but all depends on your CI/CD. How are you currently building and publishing your images? Does Elastic have access to `arm64` hardware that you will build the `arm` docker images on? Or are you planning on using `qemu` and ` binfmt_misc` to emulate the architectures so you can build all the images on `x86` hardware?

I am happy to lend a hand if you need when it comes to building multi-arch containers and building out manifests. Being at IBM, I do that quite a lot! :smile:

Are you planning on producing manifests for your `arm64` and `amd64` images? That would be really cool to see!!

One quick note with using `qemu` and ` binfmt_misc` is that there are some bugs you might run into. Images might seem like they built correctly, but when ran on the specific hardware you will run into errors. Hence why I encourage people to build on platform if they can. I am currently helping the [Jenkins Docker team]( https://github.com/jenkinsci/docker) to build on platform for each of their images. AWS does offer `arm64` resources, in case you guys were interested.